docs/examples.md

```diff
@@ -238,10 +238,32 @@ For the common case where you want to [`include`](attrs.filters.include) or [`ex
 ...     z: int
 
 >>> asdict(C("foo", "2", 3),
-...        filter=filters.include(int, fields(C).x))
+...        filter=filters.include(int, fields(C).x, "x"))
```

Comment:
same as above

Reply:
Removed. One example probably is enough.

Reply:
Fine. Add it back. More examples are always better?